src/Resource/DataMapper/DataMapperInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LAG\AdminBundle\Resource\DataMapper;
+
+use LAG\AdminBundle\Resource\Metadata\PropertyInterface;
+
+/**
+ * Map resource data to a property data. Use the property path to retrieve the property data. The property path should
+ * follow the PropertyAccess syntax. A property path set to "true" returns the resource data. A property path set to
```

Comment:
Not sure this contract is tested in `DataMapperTest`.

Reply:
This is tested in the two last items of the values provider :

```
        $data = new \stdClass();
        $data->id = 666;

        yield 'object' => [new Text(name: 'some', propertyPath: true), $data, $data];

        $data = new \stdClass();
        $data->id = 666;

        yield 'no_mapping' => [new Text(name: 'some', propertyPath: false), $data, null];
    }
```